### ISSUE TITLE
etlas build using etlas.dhall file fails inside eta docker image

### ISSUE THREAD

**Issue by jneira** [typelead/eta#914]
## Expected Behavior

Etlas build using `etlas.dhall` should work
## Actual Behavior

When doing a `etlas build` with a `etlas.dhall` file that have to access `dhall-to-etlas` prelude through http it throws;
```
↳ /"/"/root/project/dhall-eta.dhall sha256:cf1692707b5330f532768bcbaa7795ee56bf8b74431d063abb8435df77825015
  ↳ https://raw.githubusercontent.com/eta-lang/dhall-to-etlas/master/dhall/prelude.dhall sha256:47a62403bcd28107cb0a99b05fc4d8bc73f06277c422d059ba9efdd50307abbc

Error: Wrong host

↳ getProtocolByName: does not exist (no such protocol name: tcp)Exited with code 1
```
## Possible Fix

Add the needed config files and libs to eta docker image to allow etlas make http requests
## Steps to Reproduce

1. Trigger a circleci build of the project https://github.com/eta-lang/dhall-eta that uses https://github.com/eta-lang/dhall-eta/blob/master/.circleci/config.yml

## Context

I was trying to set up circlci for `dhall-eta`
## Your Environment
master eta docker image in circleci

**Comment by jneira** [comment]
For now i get to make work the build renaming `etlas.dhall` to `etlas.dhall.skip` before building.
However tests has to access dhall prelude files and it is expected to fail.

**Comment by jneira** [comment]
Thinking twice about that: etlas itself *is able* to access internet to download packages, so the calls done by dhall (either from the binary executable etlas or from java lib compiled by etlas) fail cause its uses tcp instead http (???).
Etlas tests reading dhall files with remote imports [fails too with another generic nasty error](https://circleci.com/gh/eta-lang/dhall-eta/11#tests):
```
/root/project/dist/build/eta-0.8.6.3/dhall-eta-0.1.0/t/tasty/build/tasty/tasty: line 36:  2153 Killed
```

**Comment by jneira** [comment]
Setting `JAVA_OPTS=-Xmx2G -Xms2G` as suggested in the circleci forum, solved the killed java process issue: https://circleci.com/gh/eta-lang/dhall-eta/13

**Comment by jneira** [comment]
However the dhall remote import is not fixed, the tests are not failing cause i am using dhall-lang tests (instead dhall-haskell ones) which doesnt have remote imports.

**Comment by rahulmutt** [comment]
So you are still getting 

```
Error: Wrong host

↳ getProtocolByName: does not exist (no such protocol name: tcp)Exited with code 1
```

if `etlas.dhall` is used right? `getProtocolByName` is a function in the `network` package so maybe searching its issue history may help.

**Comment by rahulmutt** [comment]
Perhaps this is relevant:

https://github.com/haskell/network/issues/221

**Comment by rahulmutt** [comment]
Yes, I think changing the image is the solution as discussed here:

https://github.com/NixOS/nixpkgs/issues/18038

and looks like @puffnfresh helped out with that.

**Comment by jneira** [comment]
@rahulmutt now `dhall-eta`build with `etlas.dhall` [works like a charm](https://circleci.com/gh/eta-lang/dhall-eta/24), thanks!
